sdk/core/azure-json/src/main/java/com/azure/json/JsonBoolean.java

```diff
@@ -0,0 +1,64 @@
+package com.azure.json;
+
+import java.io.IOException;
+
+/**
+ * Class representing the JSON boolean type
+ */
+public class JsonBoolean extends JsonElement {
+    /**
+     * Stores the String representation of the JsonBoolean object.
+     * Can only be either "true" or "false".
+     */
+    private String booleanValue;
```

Comment:
It seems like you could store the state of the boolean as a boolean, rather than as a String that you need to then keep doing string comparisons with.